### PR TITLE
gnu(8160)=8007

### PR DESCRIPTION
**Pull request by markuspf**
This took `20912.577` to construct the groups, and there were no lists to be split.

### CODE REVIEW

**Comment by olexandr-konovalov** [comment]
Thank you for submitting new gnu(n). We will recalculate it and add to the database if we get the same result.

Checklist for the reviewer:
- [x] checked and assigned labels "new (n<50000)" or "recomputation", if needed
- [x] indicated that the recomputation has been started
- [x] reproducible value of gnu(n)
- [x] no essential runtime differences
- [x] sufficient level of technical details
- [x] assigned label "replicated" after successful review

Checklist for the maintainer:
- [x] correct arguments of commands to store data
- [x] correct location of new line in the data file
- [x] mention reviewer in the commit message

**Comment by olexandr-konovalov** [comment]
gnu(8160)=8007

GAP      4.8.3
GrpConst 2.5
Runtime: 12617532 ms
Isomorphic groups eliminated

Replicated by @alex-konovalov on Ubuntu 16.04 on Azure cloud standard DS13 v2 instance (8 cores, 56 GB memory)

**Comment by alex-konovalov** [comment]
@markuspf the command is not in the right line - could you move it below and force-push please?

File: data/gnudata.g [hunk 1]
```diff
@@ -113,6 +113,7 @@ GNU_SAVE( 7938, 1550, WITH_GC );
 GNU_SAVE( 7965, 5, WITH_GC );
 GNU_SAVE( 8046, 30, WITH_GC );
 GNU_SAVE( 8073, 14, WITH_GC );
+GNU_SAVE( 8160, 8007, WITH_GC );
```
Comment (olexandr-konovalov) [inline data/gnudata.g:116]: I've fixed the location of this line myself while merging this PR

**Pull request merged**